easybuild/easyconfigs/n/nd2reader/nd2reader-3.0.6-intel-2017b-Python-2.7.14.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'Bundle'
+
+name = 'nd2reader'
+version = '3.0.6'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://soft-matter.github.io/pims'
+description = "PIMS is a lazy-loading interface to sequential data with numpy-like slicing."
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+
+dependencies = [
+    ('Python', '2.7.14'),
+    ('PIMS', '0.4.1', versionsuffix),
+]
+
+exts_defaultclass = 'PythonPackage'
+exts_filter = ("python -c 'import %(ext_name)s'", '')
+
+exts_list = [
+    ('xmltodict', '0.11.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/x/xmltodict'],
+        'checksums': ['8f8d7d40aa28d83f4109a7e8aa86e67a4df202d9538be40c0cb1d70da527b0df'],
+    }),
+    (name, version, {
+        'source_urls': ['https://pypi.python.org/packages/source/n/nd2reader'],
+        'checksums': ['3aad2a5a7aa9454f8e4e02e5acbb1f5d7d8ab4023265eb848ff5c3c7d9b6788b'],
+    }),
+]
+
+# specify that Bundle easyblock should run a full sanity check, rather than just trying to load the module
+full_sanity_check = True
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+
+modextrapaths = {'PYTHONPATH': ['lib/python%(pyshortver)s/site-packages']}
+
+moduleclass = 'lib'
```

Comment:
@boegel - did wonder whether these should be `bio`, but it does indeed look as though they're generic.

Reply:
Not `bio` imho, it's a generic library for reading image/video formats, afaik.